src/ActionsMenuItem.scss

```diff
@@ -0,0 +1,30 @@
+.ActionsMenuItem__button {
+    width: 4rem;
+    float: right;
+    border: 0;
+    background-color: inherit;
+}
+
+.ActionsMenuItem__button-label {
+    margin-left: 0.5rem;
+}
+
+.ActionsMenuItem {
+    border-top: 1px solid #cccccc;
+    clear: both;
+    margin: 0.5rem;
+    white-space: nowrap;
+}
+
+.ActionsMenuItem:first-of-type {
+    border-top: none;
+}
+
+.ActionsMenuItem__text {
```

Comment:
Since we applied padding at the ActionsMenu__option, let's provide the same amount of padding for the text as well.
`
.ActionsMenuItem__text {
    padding: 0.2rem 0;
    float: left;
}
`

Reply:
Done.